core/src/main/java/org/mapfish/print/map/tiled/wmts/WMTSLayerParam.java

```diff
@@ -75,13 +74,7 @@
     @HasDefaultValue
     public String[] dimensions;
 
-    /**
-     * The dpi of the returned images.
-     * <p></p>
-     * By default this is the OGC default DPI.
-     */
-    @HasDefaultValue
-    public double dpi = Constants.OGC_DPI;
```

Comment:
It's relay strange for me to change that, for me it's related to the WMTS capabilities than it's always in OGC_DPI... not?